frontend/common/src/tests/setup.ts

```diff
@@ -0,0 +1,14 @@
+import { toHaveNoViolations } from "jest-axe";
```

Comment:
Still seeing 'Unable to resolve path to module 'jest-axe' after running refresh_all.sh

Reply:
wth, it is def in the [package.json](https://github.com/GCTC-NTGC/gc-digital-talent/blob/a2ee8b9a61e0096b5670395f5a61f8ca239bb005/frontend/common/package.json#L113) 😖